File: backend-go/model/account.go
package model

type Account struct {
	AccountID int64  `json:"account_id" gorm:"primaryKey;autoIncrement;<-:false"`
	Name      string `json:"name"`
	Balance   int64  `json:"balance"`
	Alamat 		string `json:"alamat"`
	No_telp int64 `json:"no_telp"`
	Bank_id string `json:"bank_id"`
}

File: flutter_project/lib/model/account.dart
class Account {
  const Account({
    required this.accountId,
    required this.name,
    required this.balance,
    required this.alamat,
    required this.noTelepon,
    required this.bankId,
  });

  final int accountId;
  final String name;
  final int balance;
  final String alamat;
  final int noTelepon;
  final int bankId;

  factory Account.fromJson(Map<String, dynamic> json) {
    final data = json['data'];
    return Account(
      accountId: data['account_id'],
      name: data['name'],
      balance: data['balance'],
      alamat: data['alamat'],
      noTelepon: data['no_telp'],
      bankId: data['bank_id'],
    );
  }
}
